docs/DafnyRef/Types.md

```diff
@@ -2409,7 +2409,10 @@ A one-dimensional array of `n` `T` elements is created as follows:
 a := new T[n];
 ```
 The initial values of the array elements are arbitrary values of type
-`T`.
+`T`. They can also be initialized with the array display syntax:
```

Comment:
I don't see "array display syntax" defined anywhere in the manual, so I wouldn't introduce this name without defining it. 

Consider changing this to `They can be initialized using an ordered list of expressions enclosed in square brackets, similar to a sequence display expression. Here is an example:`